### ISSUE TITLE
Remove `store` package once CRUD-mode is removed

### ISSUE THREAD

**Issue by elevran** [clusterlink-net/clusterlink#412]
@orozery is the store indeed obsolete once CRD based management is done?
The package (and related) should be removed once no longer needed.

The removal of store and related code should also allow us to enable the `nilnil` linter #288

**Comment by orozery** [comment]
The more direct answer is:
The store is obsolete once CRUD-mode is removed.
